test/support/clusterpools.go

```diff
@@ -0,0 +1,115 @@
+package support
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"strings"
+
+	ocmsdk "github.com/openshift-online/ocm-sdk-go"
+	cmv1 "github.com/openshift-online/ocm-sdk-go/clustersmgmt/v1"
+	mapiclientset "github.com/openshift/client-go/machine/clientset/versioned"
+	"github.com/openshift/client-go/machine/listers/machine/v1beta1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/labels"
+)
+
+var (
+	ClusterID     string = os.Getenv("CLUSTERID")
+	machineClient mapiclientset.Interface
+	msLister      v1beta1.MachineSetLister
+	TestName      string = "test-instascale"
+)
+
+const (
+	namespaceToList = "openshift-machine-api"
+)
+
+func CreateOCMConnection(secret string) (*ocmsdk.Connection, error) {
+	logger, err := ocmsdk.NewGoLoggerBuilder().
+		Debug(false).
+		Build()
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "Can't build logger: %v\n", err)
+		os.Exit(1)
+	}
```

Comment:
What about returning `nil, err` here? To leave error processing to the upper layer.

Reply:
Updated